packages/tsup-configs/package.json

```diff
@@ -29,6 +30,10 @@
     "scripts": {
         "build": "tsup"
     },
+    "peerDependencies": {
+        "tsup": "*",
+        "typescript": "*"
```

Comment:
Why is typescript a tsup dependency again? you can't use tsup with a js project?

Reply:
Typescript is a peerdependency of tsup, we are forwarding that peer dependency to the project consuming the tsup config.

It's part of the initiave to make all major peer dependencies explicit rather than auto installed by PNPM to prevent versioning issues.